### ISSUE TITLE
Show Error log for each retry

### ISSUE THREAD

**Issue by baronvonbirra** [Bkucera/cypress-plugin-retries#31]
Hi there! 
We have our CI pipeline configured with _CYPRESS_RETRIES: 2_. In case the 3 of tries fail, just the last error is shown in the log; but is there a way to show all errors (the 3 errors, in our case)?

Thanks!

**Comment by Bkucera** [comment]
@baronvonbirra hi, you're talking about the terminal reporter correct?

**Comment by samjeffress** [comment]
I think it'd be useful to have some terminal output so it's we can tell which jobs are retrying

**Comment by Bkucera** [comment]
@samjeffress see #24

**Comment by baronvonbirra** [comment]
Hey @Bkucera! Yes, I mean the terminal reporter, sorry! 

The thing is, our scenarios sometimes fails (it could be a bug, it could be an issue after a change in the code, or even a timeout in the pipeline), and then they retry. But we have seen that sometimes, because of how our tests are designed, they fail in the retry because of another thing: for example, we are not expecting to see something that is going to be created later in that test, but as the 1st scenario has already created it before failing, it's already there and it fails. In this specific cases, it's not easy and first sight to spot what has really failed.

Not sure if my explanation was clear 😅 .

**Comment by Bkucera** [comment]
@baronvonbirra it could be sensible to hold onto the first error and if all retries fail, show that one instead of the last one. That way you'll most likely see the real failure.

**Comment by baronvonbirra** [comment]
@Bkucera yes, that's a good idea and totally fits our needs. Is there currently a way to achieve it? 🤔 

By the way, I just upgraded to v.1.5.0 and introduced the retry logging, and it's really nice. If we could get the error log for the 1st fail as you say, we will be reaaaally happy. 😄

**Comment by psurana0329** [comment]
upgraded the plugin retry version and attempting to add retry logging but for some reason but doing
module.exports=(on) => {
     require('cypress-plugin-retries/lib/plugin')(on);
}

shows nothing about retries on my terminal, All i see is the barrage of get and post requests

**Comment by Bkucera** [comment]
@psurana0329 this is off topic, please open an issue.

**Comment by psurana0329** [comment]
Do we know an eta for this? I am extremely interested in seeing the error for first failure too?

**Comment by gilgold** [comment]
Hey, I am using a custom reporter and emits test results to our own dashboard and we need to have it capture reties too (and group them by some unique ID if possible) - any way to enable reporting of all failed retries? that would be very helpful!

**Comment by Bkucera** [comment]
@gilgold I didn't want to hack in emitting extra events to try to keep the surface area of this plugin small.

But, if you want to hack on top of this, you can handle the `logRetry` task this plugin adds as done here https://github.com/Bkucera/cypress-plugin-retries/blob/master/lib/plugin.js

**Comment by Bkucera** [comment]
also, @gilgold off topic. please open an issue
